src/control-manager.cpp

```diff
@@ -256,7 +274,7 @@ namespace dynamicgraph
             {
               m_emergency_stop_triggered = true;
               SEND_MSG("Estimated torque "+toString(tau(i))+" > max torque "+toString(tau_max(i))+
-                       " for joint "+JointUtil::get_name_from_id(i), MSG_TYPE_ERROR);
+                       " for joint "+ m_robot->model().getJointName(i), MSG_TYPE_ERROR);
```

Comment:
Fix in commit bc5f539678009aa87d3eaa2a26e524133f5d03dd. But IMHO adding a remapping between name and id in a complex robotics application, in the long run, may just generate another set of confusion for newcomers...